index.html

```diff
@@ -255,43 +255,304 @@ <h2 class="section-title">Components</h2>
                 
 
                 
-                    <div class="icons-section" id="icons-example">
-                        <h2>Icons</h2>
+                    <div class="section icons-section" id="icons-example">
+                        <h2 class="section-title">Icons</h2>
                         
+
+    <h3 class="section-sub-title">Icon usage</h3>
+
+    
```

Comment:
yeah it's compiled, you can tell because it's ugly. And unfortunately we have to commit it for the github page to work